main.py

```diff
@@ -22,6 +23,7 @@ def main():
 
     repository = Repository(arguments.repository)
     vulnerabilities = Vulnerabilities(arguments.vulnerabilities)
+    all_files = vulnerabilities.get_all_file_names()
```

Comment:
Is this here by mistake? Why would we need the list of all files anywhere outside of the analyzer?

Reply:
Yeah, I had it here before, but I forgot to move it when I moved everything into analyzer.